s4e-backend/src/main/java/pl/cyfronet/s4e/config/SecurityConfig.java

```diff
@@ -112,6 +112,11 @@ private void authorizeRequests(ExpressionUrlAuthorizationConfigurer<HttpSecurity
                 .mvcMatchers(DELETE, prefix("/saved-views/{uuid}"))
                     .access("isAuthenticated() && @savedViewService.canDelete(#uuid, authentication)")
 
+                .mvcMatchers(POST, prefix("/report-templates")).authenticated()
```

Comment:
Why you don't check is user is zk member?

Reply:
See https://github.com/cyfronet-fid/sat4envi/pull/877#issuecomment-729529471.